### ISSUE TITLE
How the correct way to extend this AAC?

### ISSUE THREAD

**Issue by diorgesl** [pandaac/pandaac#9]
I would like to add some features.

**Comment by eklundchristopher** [comment]
The best way would be to create your own [packages](https://github.com/apolune), whether you clone my ones, or create your own from scratch is entirely up to you. If you are to create your own however, please make sure you utilise the correct [contracts](https://github.com/apolune/contracts) as they are vital for communication between packages.

The [theme](https://github.com/pandaac/theme-tibia/blob/master/composer.json#L21) is what glues everything together. By default, we're requiring the `pandaac/apolune` repository, which is a collection of all the individual Apolune packages. But you can just as easily include them one by one, which is what you'll want to do if you're going to use your own packages.

Basically, replace `"pandaac/apolune": "dev-master",` in your theme's composer.json file with all your own packages (and the [default Apolune ones](https://github.com/pandaac/apolune/blob/master/composer.json#L28-L42) if you don't intend to replace every package).

Once this is done, you'll need to run `composer update` to get rid of the old and bring in the new repositories. If you don't want to submit your packages to [packagist](https://packagist.org/), you may go ahead and use [VCS](https://getcomposer.org/doc/05-repositories.md#vcs) instead (also works with private repositories).

Hope that clarifies it!
